Core/LinkCleaner.swift

```diff
@@ -88,4 +88,42 @@ public class LinkCleaner {
         
         return nil
     }
+    
+    public func cleanTrackingParameters(initiator: URL?, url: URL?,
+                                        config: PrivacyConfiguration = PrivacyConfigurationManager.shared.privacyConfig) -> URL? {
+        guard let url = url, !isURLExcluded(url: url, config: config, feature: .trackingParameters) else { return nil }
+        if let initiator = initiator, isURLExcluded(url: initiator, config: config, feature: .trackingParameters) {
+            return nil
+        }
+        
+        guard let urlsComps = URLComponents(url: url, resolvingAgainstBaseURL: false) else {
+            return nil
+        }
+        guard let queryParams = urlsComps.queryItems, queryParams.count > 0 else {
+            return nil
```

Comment:
Since this is "clean tracking params and return clean url" type of method, shouldn't this just return url if it has no params at all - meaning it is "clean"?